src/Command/CommandBase.php

```diff
@@ -1192,22 +1192,26 @@ protected function checkForNewVersion(InputInterface $input, OutputInterface $ou
    *
    * @todo unify with consolidation/self-update and support unstable channels
    *
-   * @throws \Exception
+   * @throws \Exception|\GuzzleHttp\Exception\GuzzleException
    */
-  protected function hasUpdate() {
-    $client = $this->getUpdateClient();
-    $response = $client->get('https://api.github.com/repos/acquia/cli/releases');
-    $releases = json_decode($response->getBody());
-
-    if (!isset($releases[0])) {
-      throw new \Exception('API error - no release found at GitHub repository acquia/cli');
+  protected function hasUpdate(): bool {
+    try {
+      $client = $this->getUpdateClient();
+      // Intentionally silence and allow errors. It's ok if this fails.
+      $response = @$client->get('https://api.github.com/repos/acquia/cli/releases');
```

Comment:
I tried changing this URL to a one that doesn't exist to test out this PR. Didn't work despite wiping `vendor`, then doing a `ckc` and `cc`. Any idea how to try running into this issue?